adafruit_emc2101/__init__.py

```diff
@@ -343,7 +317,7 @@ def fan_speed(self):
         val = self._tach_read_lsb
         val |= self._tach_read_msb << 8
         if val < 1:
-            raise EMC2101BadValueException()
+            raise RuntimeError("Connection")
```

Comment:
I assume this is WIP?  If this is caused by a communication issue, it should be `OSError`.